_inc/lib/debugger/class-jetpack-cxn-test-base.php

```diff
@@ -282,12 +294,17 @@ public static function skipped_test( $name = 'Unnamed', $message = false ) {
 	 * @param string $resolution Optional. Steps to resolve.
 	 * @param string $action Optional. URL to direct users to self-resolve.
 	 * @param string $severity Optional. "critical" or "recommended" for failure stats. "good" for passing.
+	 * @param string $label Optional. The label to use instead of the test name.
+	 * @param string $action_label Optional. The label for the action url instead of default 'Resolve'.
 	 *
 	 * @return array Test results.
 	 */
-	public static function failing_test( $name, $message, $resolution = false, $action = false, $severity = 'critical' ) {
+	public static function failing_test( $name, $message, $resolution = false, $action = false, $severity = 'critical', $label = false, $action_label = 'Resolve' ) {
 		// Provide standard resolutions steps, but allow pass-through of non-standard ones.
 		switch ( $resolution ) {
+			case 'connect_jetpack':
+				$resolution = false;
```

Comment:
The resolution field is for a human-readable action. If the failing result is passing a custom one, it doesn't need to be listed here since it'll just pass through (L324 as modified).